### ISSUE TITLE
JOSS paper for the package

### ISSUE THREAD

**Issue by wang-boyu** [projectmesa/mesa-frames#90]
We discussed in one of the update meetings that there'll be a [JOSS paper](https://joss.readthedocs.io/en/latest/paper.html) for mesa-frames.

Do you mind creating a dedicated branch for it (e.g., `joss` or `paper` or any other branch name you prefer)? We can start creating PRs to that branch. It can then be merged into `main` when ready.

**Comment by adamamer20** [comment]
> We discussed in one of the update meetings that there'll be a [JOSS paper](https://joss.readthedocs.io/en/latest/paper.html) for mesa-frames.
> 
> Do you mind creating a dedicated branch for it (e.g., `joss` or `paper` or any other branch name you prefer)? We can start creating PRs to that branch. It can then be merged into `main` when ready.

That's wonderful! I'll do that.

**Comment by wang-boyu** [comment]
With #104 merged, @adamamer20 would you like to start drafting the paper?

The sections that the paper should have are (per https://joss.readthedocs.io/en/latest/paper.html) -

- [ ] list of authors
- [ ] summary
- [ ] statement of need
- [ ] acknowledgement of financial support (gsoc)
- [ ] references

And  we can have more sections for things like component structures, designs, etc (but not API documentations). The sections listed above are minimum requirements.

Once the paper is ready, we can submit it to JOSS, which has another checklist for reviewers: https://joss.readthedocs.io/en/latest/review_checklist.html. We can then work with the reviewers to get it published.

**Comment by adamamer20** [comment]
Perfect, I will start making the PRs. Who should I list as author, other than me?

**Comment by wang-boyu** [comment]
Thanks @adamamer20. We can start working on other sections while figuring out the authorship.

> Who should I list as author, other than me?

@projectmesa/maintainers any suggestions?

**Comment by EwoutH** [comment]
I think that’s up to you really, Adam. I think you can list yourself als sole author, or can choose to add mentors/reviewers as co-authors.

**Comment by jackiekazil** [comment]
+1 @EwoutH's comment
